src/formElementPlugins/listBuilder/src/helpers.ts

```diff
@@ -0,0 +1,112 @@
+import { ApplicationViewProps } from '../../types'
+import {
+  ApplicationDetails,
+  ElementState,
+  EvaluationOptions,
+  ResponseFull,
+  ResponsesByCode,
+  User,
+} from '../../../utils/types'
+import { TemplateElement, TemplateElementCategory } from '../../../utils/generated/graphql'
+import ApplicationViewWrapper from '../../ApplicationViewWrapper'
+import SummaryViewWrapper from '../../SummaryViewWrapper'
+import strings from '../constants'
+import { evaluateElements } from '../../../utils/helpers/evaluateElements'
+import { defaultEvaluatedElement } from '../../../utils/hooks/useLoadApplication'
+import { useUserState } from '../../../contexts/UserState'
+import { DisplayType, InputResponseField, ListItem, ListLayoutProps } from './types'
+
+// Formatting and Text manipulation
+export const getDefaultDisplayFormat = (inputFields: TemplateElement[]) => {
+  const displayString = inputFields.reduce(
+    (acc: string, { code, title }) => acc + `**${title}**: \${${code}}  \n`,
+    ''
+  )
+  return { title: '', subtitle: '', description: displayString }
+}
+
+export const substituteValues = (parameterisedString: string, item: ListItem) => {
+  const getValueFromCode = (_: string, $: string, code: string) => item[code]?.value?.text || ''
+  return parameterisedString.replace(/(\${)(.*?)(})/gm, getValueFromCode)
```

Comment:
Would be good to comment what is the string substitution/regex doing for future reference

Reply:
Sure. a3324c71